snapcraft/tests/test_plugin_go.py

```diff
@@ -129,21 +113,19 @@ class Options:
         plugin.build()
 
         self.run_mock.assert_has_calls([
-            mock.call(['env', 'GOPATH={}'.format(plugin._gopath),
-                       'go', 'get', '-t', '-d', './dir/...'],
-                      cwd=plugin._gopath_src),
-            mock.call(['env', 'GOPATH={}'.format(plugin._gopath),
-                       'go', 'install', './dir/...'],
-                      cwd=plugin._gopath_src),
+            mock.call(['go', 'get', '-t', '-d', './dir/...'],
+                      cwd=plugin._gopath_src, env=mock.ANY),
```

Comment:
Since you already have a test for this (`test_pull_local_sources()`) you might consider resetting the mock before you run build() and just asserting the build calls.